app/src/main/resources/queries/nearby_query.rq

```diff
@@ -39,6 +39,12 @@ SELECT
        # Get emoji
        OPTIONAL { ?classId wdt:P487 ?emoji0. }
        OPTIONAL { ?classId wdt:P279*/wdt:P487 ?emoji1. }
+
+       OPTIONAL {
+           ?wikipediaArticle   schema:about ?item ;
+                               schema:isPartOf <https://${LANG}.wikipedia.org/> .
+           SERVICE wikibase:label { bd:serviceParam wikibase:language "${LANG}" }
```

Comment:
Is this `SERVICE` line really needed?

Reply:
Here is the API: https://www.w3.org/TR/sparql11-query/
`wikibase:label` is different though, it allows one to use `$xyzLabel` to get the label of an item `$xyz`.